### ISSUE TITLE
AICamera android: cannot run normally, because there is no resize process and yuv2rgb function return negative value?

### ISSUE THREAD

**Issue by youngstu** [PaddlePaddle/Mobile#90]
Why push code that bugs exist and cannot run normally to github ?  And to mislead others  on purpose.

**Comment by Xreki** [comment]
What kind of error do you meet? We can successfully build the project, the only need is to put the paddle library and models in the right position. We provide a pre-built app, you can download through the QR code. https://github.com/PaddlePaddle/Mobile/tree/develop/Demo/Android/AICamera#quick-install

**Comment by youngstu** [comment]
I have tryied to run the given model such as face_mobilenet_160_91.paddle, pascal_mobilenet_300_66.paddle. Because the model size is different from camera frame size, the app cannot detect normally.

**Comment by youngstu** [comment]
Are you sure that you can run it accurately?

**Comment by youngstu** [comment]
Yeah, I can also build the project successfully but it can not detect anything.

**Comment by youngstu** [comment]
After I read the project source code, I find that there is no resize process that deals with the difference in size between camera frame size and the model input size.

**Comment by youngstu** [comment]
The pre-build app that i have tryied, it also failed to detect anying.

**Comment by Xreki** [comment]
> After I read the project source code, I find that there is no resize process that deals with the difference in size between camera frame size and the model input size.

`resize` is done in c++ codes.

**Comment by youngstu** [comment]
vivo x9s looks like that it can not detect accurately and help to fix it. thank you.

**Comment by youngstu** [comment]
Can you help to give the right position to put model ?

**Comment by Xreki** [comment]
Models should be put under `app/src/main/assets/models`.
